cmd/sentinel/sentinel.go

```diff
@@ -496,6 +496,15 @@ func (s *Sentinel) updateKeepersState(keepersState cluster.KeepersState, keepers
 		k.Healthy = s.isKeeperHealthy(k)
 	}
 
+	// Delete unhealthy keepers, except of master
```

Comment:
I'm not so sure about this change.
Will the keeper be re-added if it becomes healthy again?

Reply:
> Will the keeper be re-added if it becomes healthy again?
Any keepers will be added, because adding keepers is a job for keeper, not sentinel. And no matter is it healthy or not. I specifically tested situation with reloading standby, keeper was deleted from clusterview and then added again + replication slot was deleted and added.